src/zenml/zen_server/zen_server_api.py

```diff
@@ -29,6 +29,7 @@
 from zenml.analytics import source_context
 from zenml.constants import API, HEALTH
 from zenml.enums import AuthScheme, SourceContextTypes
+from zenml.events.base_event_flavor import events_router
```

Comment:
Is this really meant to automatically populate the REST API routers based on the available event source plugins ? or just a temporary hacky convenience ?

Reply:
temporary hacky convenience